src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -1,509 +1,104 @@
-      return res.status(404).json({
-        message: `Proposed solution with id #${psId} not found for change request #${crId}`
-      });
-    }
-    // update proposed solution
-    await prisma.proposed_Solution.update({
-      where: { proposedSolutionId: psId },
-      data: {
-        approved: true
-      }
-    });
-    const wbs = await prisma.wBS_Element.findUnique({
-      where: {
-        wbsElementId: foundCR.wbsElementId
-      },
-      include: {
-        workPackage: true,
-        project: true
-      }
-    });
-    if (!wbs) {
-      return res.status(404).json({ message: `WBS element with id #${foundCR.wbsElementId} not found` });
-    }
-    const { workPackage, project } = wbs;
-
-    if (!workPackage && project) {
-      const newBudget = project.budget + foundPs.budgetImpact;
-      const change = {
-        changeRequestId: crId,
-        implementerId: reviewerId,
-        detail: buildChangeDetail('Budget', String(project.budget), String(newBudget))
-      };
-      await prisma.project.update({
-        where: { projectId: project.projectId },
-        data: {
-          budget: newBudget,
-          wbsElement: {
-            update: {
-              changes: {
-                create: change
-              }
-            }
-          }
-        }
-      });
-    } else if (workPackage) {
-      const wpProj = await prisma.project.findUnique({
-        where: { projectId: workPackage.projectId }
-      });
-      if (!wpProj) {
-        return res.status(404).json({ message: 'Work package project not found' });
-      }
-      const newBudget = wpProj.budget + foundPs.budgetImpact;
-      const updatedDuration = workPackage.duration + foundPs.timelineImpact;
-
-      const changes = [
-        {
-          changeRequestId: crId,
-          implementerId: reviewerId,
-          detail: buildChangeDetail('Budget', String(wpProj.budget), String(newBudget))
-        },
-        {
-          changeRequestId: crId,
-          implementerId: reviewerId,
-          detail: buildChangeDetail('Duration', String(workPackage.duration), String(updatedDuration))
-        }
-      ];
-      await prisma.project.update({
-        where: { projectId: workPackage.projectId },
-        data: {
-          budget: newBudget,
-          workPackages: {
-            update: {
-              where: { workPackageId: workPackage.workPackageId },
-              data: {
-                duration: updatedDuration,
-                wbsElement: {
-                  update: {
-                    changes: {
-                      create: changes[1]
-                    }
-                  }
-                }
-              }
-            }
-          },
-          wbsElement: {
-            update: {
-              changes: {
-                create: changes[0]
-              }
-            }
-          }
-        }
-      });
+import { Request, Response, NextFunction } from 'express';
```

Comment:
new line between each static function

also you can take out all the one line comments, it's the service class that should be heavily documented